components/automate-ui/src/app/services/event-feed/event-feed.service.ts

```diff
@@ -25,11 +25,12 @@ import { environment } from '../../../environments/environment';
 import * as moment from 'moment/moment';
 const GATEWAY_URL = environment.gateway_url;
 const CONFIG_MGMT_URL = environment.config_mgmt_url;
-const ENTITY_TYPE_TAG = 'entity_type';
+const ENTITY_TYPE_TAG = 'event-type';
```

Comment:
This was an existing bug that prevented the "version" event-types from being added in the filters.

Reply:
This also fixed the same problem for data bag items.